spec/index.bs

```diff
@@ -6157,11 +6163,16 @@ dictionary GPURenderPassDepthStencilAttachment {
 
     : <dfn>depthLoadValue</dfn>
     ::
-        If a {{GPULoadOp}}, indicates the load operation to perform on
-        {{GPURenderPassDepthStencilAttachment/view}}'s depth component prior to
-        executing the render pass.
-        If a `float`, indicates the value to clear {{GPURenderPassDepthStencilAttachment/view}}'s
-        depth component to prior to executing the render pass.
+        If {{GPULoadOp/"load"}}, indicates that {{GPURenderPassDepthStencilAttachment/view}}'s
+        depth aspect contents should be preserved prior to executing the render pass,
+        and loaded from memory into the render pass.
+
+        Otherwise, indicates that {{GPURenderPassDepthStencilAttachment/view}}'s
```

Comment:
that's going to play well with reverse-Z ;)

Reply:
heh, I forgot to think about that (have thought about it in the past). If we used the word "clear" instead then it would be ambiguous whether it clears to 1.0. So in this case "zero" seems much less confusing.

We could also split the GPULoadOp enum for depth/stencil. The "zero" shorthand is not really a shorthand for depth/stencil clear values.

Reply:
Typing "1.0" is unambiguous and short, I don't think there is a real need for a separate `GPUDepthLoadOp`